entc/gen/feature.go

```diff
@@ -43,11 +43,25 @@ var (
 		},
 	}
 
+	// FeatureCustomSchemas allows users to pass init time alternate schema names
+	// for each ent model. This is useful if your SQL tables are spread out against
+	// multiple databases.
+	FeatureCustomSchemas = Feature{
+		Name:        "custom-schemas",
+		Stage:       Experimental,
+		Default:     false,
+		Description: "Allows alternate schema names for each ent model. Useful if SQL tables are spread out against multiple databases",
+		cleanup: func(c *Config) error {
+			return os.RemoveAll(filepath.Join(c.Target, "dbschema"))
+		},
+	}
+
```

Comment:
Great, but let's move the name to be prefixed with `"sql"` (`"sql/<Name>"`), and maybe propose other names to it 😃  (e.g. `"sql/dynamicschema"`, `"sql/schemaconfig"`, `"sql/alternate-schema"`, etc).

I like the `"sql/schemaconfig"` one, because as I see it, this feautre-flag exposes a configuration functionality for the generated assets.

Reply:
@a8m sounds good 👍🏼 should the `dbschema` package be instead named `schemaconfig` as well to stay consistent?

Reply:
@a8m updated this PR by renaming the feature to `sql/schemaconfig`